### ISSUE TITLE
Improve consistency of labels of ABC transporter activities

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#17164]
Hello, 

ATP-coupled transporters are named in several different ways: 

- ATPase activity, coupled to movement of substances
- phospholipid-translocating ATPase activity
- ATPase-coupled anion transmembrane transporter activity
- vitamin B12 ABC transporter activity
(and maybe other ways). 

I will rename all these terms "ATPase-coupled transmembrane transporter activity"
and "ATP-coupled transmembrane [x] transporter activity" for all children. 

Thanks, Pascale

**Comment by pgaudet** [comment]
@amandamackie Does thsi term label look ok to you ?

**Comment by amandamackie** [comment]
Hi Pascale, I'm out of the office for next two weeks. Will look into this when I return. Amanda

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by vanaukenk** [comment]
@pgaudet - I have a question about GO:0004012, phospholipid-translocating ATPase activity.

Is that activity transmembrane transport, or does it refer to translocation of phospholipids from one monolayer to the other, e.g. cytoplasmic leaflet to exoplasmic leaflet?

**Comment by pgaudet** [comment]
Hi @vanaukenk 
Good point. These are described as flippases or scramblases, see https://en.wikipedia.org/wiki/ATP10A "The aminophospholipid translocases transport phosphatidylserine and phosphatidylethanolamine from one side of a bilayer to another."

I will not touch these for now - these will probably need an annotation review. 

Thanks, Pascale

**Comment by ValWood** [comment]
should be placed under
GO:0120013    intermembrane lipid transfer activity
or GO:0120013   intermembrane lipid transfer activity
which  should also have a common parent, probably.

**Comment by ValWood** [comment]
corrected above to
GO:0140303    intramembrane lipid transporter activity

but 
phospholipid-translocating ATPase activity (GO:0004012)
appear to be defined as  GO:0140303 intramembrane lipid transporter activity (inner to outer leaflet)

possibly equivalent?

the intermembrane ones are the 
sec14 cytosolic factor family

**Comment by amandamackie** [comment]
I have a couple of comments:

Transport ATPases are generally classified into 3 or 4 types: ABC (for ATP-binding cassette) transporters; P-type (they form a phosphorylated intermediate as part of the transport mechanism; see GO:0015662) and the F-type and V-type (the proton pumps; ?GO:0044769). Does this distinction need to be covered/preserved in molecular function terms? A good starting reference is PMID16691464

I noticed that the current term GO:0042626 (ATPase activity coupled to transmembrane movement of substances) is a child of GO:0015405 (P-P bond hydrolysis driven transmembrane transporter activity). This latter definition is misleading; it is the hydrolysis of phosphoanhydride bonds in ATP that provides energy for transport, not P-P bonds.

**Comment by pgaudet** [comment]
Thanks @amandamackie 
I dont think the parent term 'P-P bond hydrolysis driven transmembrane transporter activity' is needed anyway; I think it was intended for proteins that use another nucleotide as a source of energy, but I don't think this exists; does it ?

**Comment by pgaudet** [comment]
Changed label from 'ATPase activity, coupled to transmembrane movement of substances' to 'ATPase-coupled transmembrane transporter activity' 
added synonyms: 
+synonym: "ATP-coupled transmembrane transporter activity" EXACT []
+synonym: "ATP-dependent transmembrane transporter activity" EXACT []
+synonym: "ATPase activity, coupled to transmembrane movement of substances" EXACT []

Changed subclass from 'is a' ATPase activity, coupled to movement of substances
to has_part GO:0016887 ! ATPase activity

@ukemi @cmungall Can I merge this (ie using 'has part' for compound functions') ? 
Pascale

**Comment by deustp01** [comment]
> Transport ATPases are generally classified into 3 or 4 types: ABC (for ATP-binding cassette) transporters; P-type (they form a phosphorylated intermediate as part of the transport mechanism; see GO:0015662) and the F-type and V-type (the proton pumps; ?GO:0044769). Does this distinction need to be covered/preserved in molecular function terms? 

Three or four different molecular mechanisms should be three or four different molecular functions, right? Especially as each is enabled by a large and distinct family of proteins.

**Comment by pgaudet** [comment]
Ontology call: I will go ahead with this branch (using 'has part' for ATPase activity of the ABC transporter, and 'transmembrane transporter activity' as the genera. )

**Comment by amandamackie** [comment]
> 
> 
> Thanks @amandamackie
> I dont think the parent term 'P-P bond hydrolysis driven transmembrane transporter activity' is needed anyway; I think it was intended for proteins that use another nucleotide as a source of energy, but I don't think this exists; does it ?

I think GTP can be used as a source of energy for transport but my point was more that there are no P-P (ie phosphorous-phosphorous) bonds in ATP or any other nucleotide, the high energy bonds are phosphanhydride bonds which are the bonds between two phosphate molecules. I guess it's a non-issue if the term is being removed.

**Comment by pgaudet** [comment]
https://github.com/geneontology/go-ontology/pull/17220

**Comment by pgaudet** [comment]
I also need to check EC cross references. 
@hdrabkin I see that some of the ECs you've added from pz; was this some time ago ? Many of these have new ECs (7.x.x.x). Were you planning to change these? (I can do it; I just don't want to duplicate work). 

Thanks, Pascale

**Comment by hdrabkin** [comment]
Yes, that is why I put the EC:7.x terms in for parents;
But, I have NOT changed ones that I could not assign correct RHEA to (usually due to specificity issues, etc.
However, I can go in and just change the EC numbers.
Let me go through my spreadsheet for these

**Comment by hdrabkin** [comment]
I see 7 on my list; all of these had rhea issues so I did not edit the term but I can go in and just change the EC right now. This is why I added the EC:7.n parent classes because the is_a will need to change to map to the new EC class. 
example

GO:0102017 | ATPase-coupled alkylphosphonate   transmembrane transporter activity | EC:3.6.3.28 
transferred   entry: 7.3.2.2 |

**Comment by hdrabkin** [comment]
Ah; found another "L-arginine-importing   ATPase activity
EC:3.6.3.21
Transferred   entry: 7.4.2.1
Will do this under my issue for the rhea stuff

**Comment by pgaudet** [comment]
Great ! I have been using the redirects from EC; I hope these are OK.

**Comment by pgaudet** [comment]
@hdrabkin 
WRT 'L-arginine-importing ATPase activity': I was going to propose to obsolete these (or merge into the parent), since the 'importing' part is a process, not a function. 

Can I add this as another xref to the parent term even if it's not the exact MF ? In other words, are we trying to be exhaustive about xrefs to EC ? 

Thanks, Pascale

**Comment by deustp01** [comment]
> WRT 'L-arginine-importing ATPase activity': I was going to propose to obsolete these (or merge into the parent), since the 'importing' part is a process, not a function.

Hmm. Is the word "importing" used here to indicate the overall process of arginine uptake, or is it used narrowly to indicate the direction of the arginine transport activity (into the cell across the plasma membrane)?

**Comment by hdrabkin** [comment]
Yes; but make sure if you do that you change the is_a parent; Tomorrow I'll add the 3 digit EC for class 7 (I already added the 1 and 2 digit. I

**Comment by hdrabkin** [comment]
7.4.2.1 ABC-type polar-amino-acid transporter
I think it should be fine as parent is arginine-importing ATPase activity, and arginine is polar.

**Comment by pgaudet** [comment]
> Is the word "importing" used here to indicate the overall process of arginine uptake, or is it used narrowly to indicate the direction of the arginine transport activity (into the cell across the plasma membrane)?

What would be the difference ? Isn't that a process regardless?

**Comment by hdrabkin** [comment]
I would think import does imply a direction.  The EC def "ATP + H2O + polar amino acid-[polar amino acid-binding protein][side 1] = ADP + phosphate + polar amino acid[side 2] + [polar amino acid-binding protein][side 1] " doesn't specifiy what side is outside or in. However, the RHEA is very explicit
"ATP + H2O + L-arginine(out) = ADP + H(+) + L-arginine(in) + phosphate
Cross reference: EC 7.4.2.1
So I would not remove the import from the term name

**Comment by deustp01** [comment]
It's not a process because the transporter in a single step mediates the translocation of a molecule of arginine inwards across the plasma membrane simultaneously with the hydrolysis of a molecule of ATP, so no more a process than any other ATP-dependent transport activity.

**Comment by pgaudet** [comment]
If it's not a process then most 'import' and 'export' terms must be removed from Process fo eg: 

For eg children of  GO:0098739   import across plasma membrane

![image](https://user-images.githubusercontent.com/4782928/57304616-ef850180-70df-11e9-870a-6f3706e4861a.png)

Most are single steps....

**Comment by deustp01** [comment]
If by definition processes cannot be single steps (maybe not a universally popular definition), then as you say most of the terms on the list are functions, not processes.

**Comment by hdrabkin** [comment]
Just looking; if you merge the L-arginine importing into L-arginine transmembrane transporter, the EC might map (inside/outside, but the RHEA would no longer map correct because it specifies direction.

**Comment by ValWood** [comment]
But many processes are the sum of many different transporters.

"Glucose import" for example can be performed by many different gene products of different affinities high, low, of expressed under certain conditions. The process of glucose import does therefore have multiple steps they just happen to be performed (largely) by gene products doing the same MF.

Import and export are also regulated by different pathways, so it is important to distinguis at the process level.

So importer is a function, but import is a process.

My preference would be to have import as a process, because helps us when describing processes to know the directionality of transport.

I guess its an arbitrary decision and we could have both MF and BP, but I would really like to keep import  and export biological processes  for modularization.  For example import and export  into the vacuole /mitochondria/ across plasma membrane are differently aligned with other processes. For many things if we only capture as "transport" we lose a lot at the process level.


Also, we can easily  couple the basic "transporter" MF  to the import process to define the specificity
here is a recent  nice example paper using zinc:
https://www.pombase.org/reference/PMID:29529046

**Comment by ValWood** [comment]
Please don't remove import/export from process...;(

**Comment by ValWood** [comment]
If we did do this this, to get the specificity in the zinc example above we would need to do

for
zinc ion transmembrane transporter activity- >

would need
zinc ion organellar transmembrane importer activity
zinc ion vacuolar transmembrane importer activity
etc

it would explode the MF branch but would not gain anything.....

and everyone who has done merges would need to reannotate again

**Comment by ValWood** [comment]
@ukemi phew!

**Comment by ukemi** [comment]
PMID:23908456

**Comment by pgaudet** [comment]
@ValWood Right, I was hoping to keep transmembrane transporter activity in MF and import/export in BP (this is what you and I discussed, as far as I remember).

But I wanted to know how @deustp01 was envisioning the split. 

Pascale

**Comment by deustp01** [comment]
PMID:23908456 from @ukemi illustrates the split really nicely for glutamine: glutamine transport is an activity mediated by a specific gene product or gene products in the plasma membrane of specific kinds of kidney cells, but it occurs as one part of a larger process (details of the other parts and their interconnections given in the paper), in this case tightly integrating several transport and metabolic activities to maintain pH.

I guess that a simpler but similar story could be told for arginine: there's a transporter activity (in this case, an ATP-dependent one), but in normal cells its function will always be part of a larger process in which other activities move counter-ions and other stuff to maintain charge balance and pH, whether or not we have the experimental data right now to make that complete process annotation.

Does that help?

**Comment by pgaudet** [comment]
I think so - but import versus export should be kept under BP; right ? 
In any case - L-arginine-importing ATPase activity is now under https://enzyme.expasy.org/EC/7.4.2.1

The Accepted Name: is ABC-type polar-amino-acid transporter.
Alternative Name(s):  Histidine permease.
Polar-amino-acid-transporting ATPase.


So a merge does seem appropriate. 

Pascale

**Comment by hdrabkin** [comment]
Ah; I see RHEA:14674
a polar amino acid(out) + ATP + H2O => a polar amino acid(in) + ADP + H(+) + phosphate
So may be ok.

**Comment by pgaudet** [comment]
Merge the following 2 terms (no annotations, no references): 
'2-aminoethylphosphonate transmembrane transporter activity'  -> no xref
merge into 
'ATPase-coupled 2-aminoethylphosphonate transporter activity' -> RHEA:32775
This is the only reaction in RHEA that involves 2-aminoethylphosphonate, suggesting this is the only type of transporter.

**Comment by pgaudet** [comment]
Obsoleted xenobiotic transporters (no annotations; and this is not a function, since the xenobiotic transporters are no specific): 

 id: GO:0103113
-name: glucosyl-oleandomycin-exporting ATPase activity
+name: obsolete glucosyl-oleandomycin-exporting ATPase activity

 id: GO:0043216
-name: ATPase-coupled daunorubicin transmembrane transporter activity
+name: obsolete ATPase-coupled daunorubicin transmembrane transporter activity

**Comment by pgaudet** [comment]
Merge 
 GO:0015623 ATPase-coupled iron-chelate transporter activity -> 0 annotations
into its child
GO:0015625 ATPase-coupled ferric-hydroxamate transmembrane transporter activity  -> 2 EXP

both share the same EC, EC 7.2.2.16, that corresponds to GO:0015625.

**Comment by pgaudet** [comment]
Obsolete 
GO:0042969 lactone transport
GO:0042971 lactone transmembrane transporter activity
no annotations, no references, no xrefs.

**Comment by pgaudet** [comment]
Merge 
GO:0015604 organic phosphonate transmembrane transporter activity -> 3 IEAs, to ABC transporters (IPR005769 + IPR005770) 
into GO:0015416 ATPase-coupled organic phosphonate transmembrane transporter activity 

@sarach06 @EBI-Hsinyu Does InterPro automatically map merged terms ? 

Thanks, Pascale

**Comment by pgaudet** [comment]
And, merge corresponding processes: 

GO:0042916 alkylphosphonate transport -> 0 annotation
into parent
GO:0015716 organic phosphonate transport

**Comment by pgaudet** [comment]
Merge GO:0070812 ATPase-coupled glycerol-2-phosphate transmembrane transporter activity -> 0 annotations, no references
into GO:0015430 ATPase-coupled glycerol-3-phosphate transmembrane transporter activity 

This is the same transporter, that can also transport G2P, see PMID:19429609

**Comment by pgaudet** [comment]
Merge 

GO:0015594 ATPase-coupled putrescine transmembrane transporter activity -> 0 annotations
GO:0015595 ATPase-coupled spermidine transmembrane transporter activity  -> 4 EXP 
into GO:0015417 ATPase-coupled polyamine transmembrane transporter activity -> 0 annotations

Same EC, same Rhea - the transporters are able to transport  all those substrates.

**Comment by pgaudet** [comment]
*    GO:0015086 cadmium ion transmembrane transporter activity => 21 EXP, ABC-type, SLC
*  GO:0015434 ATPase-coupled cadmium transmembrane transporter activity => 1 EXP annotation, TAIR HMA1 (AT4G37270), a P-type
*  GO:0008551   cadmium transmembrane transporter activity, phosphorylative mechanism => 3 EXP annotations to P-types

 - [ ] Note: RHEA:12132 corresponds to both ABC-type and P-type (currently linked to P-type). Emailed RHEA helpdesk.

**Comment by pgaudet** [comment]
Merge 
GO:0048474 D-methionine transmembrane transporter activity
into 
GO:0033232 ATPase-coupled D-methionine transporter activity 

I cannot find an example of a D-methionine transporter that is not an ABC transporter. (Also, it seems like they are all bacterial)

**Comment by pgaudet** [comment]
Obsolete GO:0033220 ATPase-coupled amide-transporter activity

-> unnecessary grouping class.

**Comment by pgaudet** [comment]
Merge 
GO:0033155 oligogalacturonide transmembrane transporter activity
into  GO:0033154 ATPase-coupled oligogalacturonide transmembrane transporter activity and rename 'ABC-type'

Cannot find evidence for a non-ABC-type
